docs/DeveloperGuide.md

```diff
@@ -29,6 +29,7 @@ This guide is for developers looking to modify GULIO. For users of GULIO, please
     * [Adding of Lesson](#adding-of-lesson)
     * [Adding of Cheat-Sheet](#adding-of-cheat-sheet)
     * [Loading & Storing of Data](#loading--storing-of-data)
+    * Design Considerations
```

Comment:
I think the link is missing here

Reply:
I was not sure how to add the link without getting the tag from Github, adding it in my next PR. Thanks!